app/views/events/forms/_Tournament.html.erb

```diff
@@ -1,8 +1,19 @@
-<div class="form-group">
+<%= render "events/forms/Dates", f: form%>
+
+<div class="form-group" id="event_player_types">
+  <%= form.label :player_type, :class => 'control-label col-lg-2' %>
+  <div class="col-lg-10">
+    <%= form.collection_select :player_type, event_player_types_localize, :first, :second,
+                            {selected: @event.player_type, include_blank: t('events.new.select_player_type')},
+                            { :class => 'form-control', :id => 'event_player_type'}%>
+  </div>
+</div>
+
+<div class="form-group" id="event_game_modes">
   <%= form.label :game_mode, class: 'control-label col-lg-2' %>
   <div class="col-lg-10">
-    <%= form.select(:game_mode, Tournament.game_modes.map {|k, v| [k.humanize, v]},
-                 { include_blank: t('events.new.select_game_mode')},
-                 { class: 'form-control' }) %>
+    <%= form.collection_select :game_mode, tournament_gamemodes_localize, :first, :second,
+                            {selected: @event.game_mode, include_blank: t('events.new.select_game_mode')},
+                            { :class => 'form-control', id: :tournament_game_mode}%>
```

Comment:
There are a few cases where oldschool hashing syntax is used with :key => "value" instead of key: "value". Maybe just let rubocop fix this if it's possible to use in views?